Code/GraphMol/MolPickler.h

```diff
@@ -138,6 +138,8 @@ class RDKIT_GRAPHMOL_EXPORT MolPickler {
     BEGINCONFPROPS,
     BEGINCONFS_DOUBLE,
     QUERY_TYPELABEL,
+    // add new entries above here
+    INVALID_TAG = 255
```

Comment:
I think it would be better not to explicitly make this 255, then it would be one more than QUERY_TYPELABEL and would be a better invalid tag check.

Reply:
we already do the invalid tag check when we're processing the tags. The goal here was to avoid the undefined behavior that you otherwise get if a badly constructed pkl contains a tag with a value that is larger than the largest possible in the enum.
We read the tags from the stream into a char, so with INVALID_TAG set to 255 we know that the `static_cast<Tags>(tmpChar)` will never overflow